tests/test_device.py

```diff
@@ -71,15 +71,16 @@ def test_query_gid(self):
                 ctx.query_gid(port_num=1, index=0)
 
     @staticmethod
-    def verify_device_attr(attr):
+    def verify_device_attr(attr, device):
         """
         Helper method that verifies correctness of some members of DeviceAttr
         object.
         :param attr: A DeviceAttr object
```

Comment:
please add the device param to the docstring

Reply:
Ack